styles/components/_button.scss

```diff
@@ -0,0 +1,2 @@
+.button {
+}
```

Comment:
Why is it empty?

Reply:
It is prepared for the future style definitions from the client